### ISSUE TITLE
Feature idea: Autotune

### ISSUE THREAD

**Issue by bdbabiak** [ctubio/Krypto-trading-bot#738]
If one looks at the percent profitability on the UI %-asset-prof and %-currency-prof, and if we have the tb%, we are trying to maximize [%currency-prof - (-100*%-asset-prof/%-tb)] if your %-tb is 50%, then %-currency-prof - 2 * %-asset-prof, if this works and especially if %-currency-prof is always 0 or positive, then the bot is handly beating buy and hold. But playing with settings, it was easy to do this. I would like to make an autotune button that does this automatically with a genetic algorithm.

**Comment by Camille92** [comment]
That would be awesome!

**Comment by mistacoda** [comment]
we could merge it into separate tool https://github.com/ctubio/Krypto-trading-bot/issues/796

what you think @bdbabiak ?

**Comment by halessi** [comment]
Was any progress ever made on this?

**Comment by bdbabiak** [comment]
No, I wandered off to other projects right now. But I'll be back. But I would have to add Binance first to make it worth my while.
